formats/build.sbt

```diff
@@ -5,6 +5,7 @@ libraryDependencies ++= Seq(
   "org.scala-lang.modules" %% "scala-xml" % "2.1.0",
   ("com.github.pathikrit" %% "better-files" % "3.9.1" % Test).cross(CrossVersion.for3Use2_13),
   "com.github.scopt" %% "scopt" % "4.1.0",
+  "io.spray" %% "spray-json" % "1.3.6" // same JSON parser as solidity2cpg
```

Comment:
```suggestion
  "io.spray" %% "spray-json" % "1.3.6"
```

Reply:
probably confusing to others, given that solidity2cpg is in a different repo etc.